### ISSUE TITLE
--node=system won't work if there's already a compiled node on the virtualenv

### ISSUE THREAD

**Issue by maniqui** [ekalinin/nodeenv#43]
This may not be a bug per-se, but I've found this: on a virtualenv, if you first run `nodeenv -p`, it will download, compile and install a copy of `node` on `~/.virtualenvs/myproject/bin`. So far, so good.

But later, if you decide to use the system-wide `node` install, running `nodeenv -p --node=system` won't help you (nor running `nodeenv--node=system`, without the `-p` option).
The standard output will tell you that some changes are applied to `~/.virtualenvs/myproject/bin/activate` and it may even install npm, but the compiled `node` will be still there in `~/.virtualenvs/myproject/bin/`, and a `which node` will tell you that that's the `node` that will be used.

In other words, the `--node=system` didn't take any effect, and one could say that it "failed silently".

BTW, as a side-effect of running `nodeenv -p` a few times, I noticed that the same lines are n-plicated inside `~/.virtualenvs/myproject/bin/activate`. I don't know if there is a "cure" for that, but just letting you know. I manually cleaned the n-plicated lines.
